### ISSUE TITLE
Error when "Read All"

### ISSUE THREAD

**Issue by danielschill** [Xjph/ObservatoryCore#50]
Throws following error when doing a read all. Other works fine.

`Error in Observatory Explorer while handling Scan: Sequence contains no elements`

```
The JSON value could not be converted to Observatory.Framework.Files.ParameterTypes.CrimeType. Path: $.CrimeType | LineNumber: 0 | BytePositionInLine: 92.
File: Journal.210828221831.01.log
Line: { "timestamp":"2021-08-29T07:54:13Z", "event":"CommitCrime", "CrimeType":"onFoot_eBreachUse", "Faction":"Alliance of Mechagii", "Fine":250 }
```

**Comment by Xjph** [comment]
Your two errors aren't the same thing. I can fix the second pretty easily and will do so in the next release (missing crime type), but I'm not sure what's causing the first without more detail. Would you be able to provide the scan that's causing it, or a copy of your journal files?

**Comment by Xjph** [comment]
Resolved in [v0.2.22094.2356](https://github.com/Xjph/ObservatoryCore/tree/v0.2.22094.2356)
